src/SFA.DAS.CaptureTheFlag.Web/Views/Shared/_LoginPartial.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@using Microsoft.AspNetCore.Identity
```

Comment:
The styling on this goes a bit naff once you've clicked the links once as you end up with Purple on black in the header which is not a great combination